CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## 4.2.0
+
+This is a major Teleport release with a focus on new features and bug fixes.
+
+### Improvements
+
+* Alpha: Enhanced Session Recording let's you know what's really happening during a Teleport Session. [Read the docs](https://gravitational.com/teleport/docs/ver/4.2/features/enhanced_session_recording/).
+* Alpha: Workflows API lets admins escalate RBAC roles. [Read the docs](https://gravitational.com/teleport/docs/ver/4.2/enterprise/#approval-workflows).
```

Comment:
`Workflows API lets admins escalate RBAC roles in response to user requests.`

Reply:
Corrected in f62c4620dd565a3894a900ab435f11a991e5e9e6